.github/workflows/unit-tests.yml

```diff
@@ -35,15 +35,15 @@ jobs:
       - run: composer install --no-progress --no-suggest --prefer-dist --optimize-autoloader
 
       - name: Install dependencies and SQL
-        run: |
-          mkdir -p coverage
-          apk add npm
-          npm install -g less clean-css uglify-js
-          mysql -uroot -ppassword -h mysql -e 'CREATE DATABASE atk4;'
-          mysql -uroot -ppassword -h mysql atk4 < demos/atk4.sql
-
-          mv demos/db.github.php demos/db.php
-          (cd js; npm install; npm run build)
+        run:
```

Comment:
This newly breaks GH tests. The Travis is already fixed by PR #931. This PR should be completely closed as it does not provide anything at this stage.